src/main/java/com/cognifide/aemrules/htl/lex/ElementTokenizer.java

```diff
@@ -33,11 +32,11 @@
 
 class ElementTokenizer extends AbstractTokenizer<List<Node>> {
 
-    private static EndQNameMatcher endQNameMatcher = new EndQNameMatcher();
+    private static final EndQNameMatcher endQNameMatcher = new EndQNameMatcher();
```

Comment:
Since this is `private`, `static` _and_ `final`, it should follow the naming convention for constants, i.e. `END_Q_NAME_MATCHER`. Same below.

Reply:
my change is not about fixing all code style issues, this was highlighted by intellij and fixable by auto-fix so I just clicked it

Reply:
fixed